packages/react-native/src/evm/wallets/connectors/embedded-wallet/embedded-connector.ts

```diff
@@ -59,6 +59,9 @@ export class EmbeddedWalletConnector extends Connector<EmbeddedWalletConnectionA
       await this.switchChain(options.chainId);
     }
 
+    if (this.email) {
+      saveConnectedEmail(this.email);
```

Comment:
was this needed?

Reply:
I was doing it on sendverification but at that point the wallet is not connected so doing it after a successful connection.

I'm saving the email here because on auto-connection the email is not in the connect params and we want to show it in the UI

Reply:
sg